CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - Fixed [#649](https://github.com/JabRef/jabref/issues/649): Key bindings are now working in the preview panel
 - Fixed [#410](https://github.com/JabRef/jabref/issues/410): Find unlinked files no longer freezes when extracting entry from PDF content
 - Fixed [#936](https://github.com/JabRef/jabref/issues/936): Preview panel is now updated when an entry is cut/deleted
+- Fixed Bug introduced with Commit [b3aee00](https://github.com/JabRef/jabref/commit/b3aee00bae8b589aa1252ef182f8b3293d06bd69): Entry is now correctly removed from BibDatabase  
```

Comment:
Yes, will fix #991 (just tested it, I overlooked that issue, thanks for mentioning).
I'll adjust the Changelog.